src/olympia/amo/models.py

```diff
@@ -433,6 +433,9 @@ def _cache_key(cls, pk, db):
         key_parts = ('o', cls._meta, pk, 'default')
         return ':'.join(map(force_text, key_parts))
 
+    def serializable_reference(self):
```

Comment:
stick this in `UncachedModelBase` instead for `Collection`

Reply:
Yup